core/templates/dev/head/pages/exploration_editor/editor_tab/StateHints.js

```diff
@@ -115,10 +117,13 @@ oppia.controller('StateHints', [
             $scope.hintIndex = stateHintsService.displayed.length + 1;
 
             $scope.saveHint = function() {
+              var contentId = GenerateContentIdService.generateUniqueId(
```

Comment:
Maybe instead of generateUniqueId, we should call it something more like getNextId? Unique Id makes it sound like it's a random string. Up to you though I'm ultimately okay with either.

Reply:
Done!